test/functional/checkupdate_v2/chk-update-slow-server.bats

```diff
@@ -2,24 +2,27 @@
 
 load "../testlib"
 
-server_pid=""
-port=""
-THEME_DIRNAME="$FUNC_DIR/checkupdate_v2"
+test_setup() {
 
-test_teardown() {
+	create_test_environment "$TEST_NAME"
 
-	echo "terminating web server..." >&3
-	kill "$server_pid" >&3
-	destroy_test_environment "$TEST_NAME"
+	# start slow response web server
+	start_web_server -s
 
+	# update PORT environment variable
+	set_env_variables "$TEST_NAME"
```

Comment:
this function is already run from the create_test_environment function.

Reply:
I added the PORT environment variable to set_env_variables which isn't known until after the call to start_web_server. I can remove the call to set_env_variables by moving the code from test_setup to global_setup and creating a test_setup that does nothing.

Reply:
Now, I think it's more clear to remove the call to set_env_variables, move the setup/teardown to the global_setup/global_teardown functions, and do nothing with test_setup/test_teardown. I plan on making the change here.